github.css

```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```

Comment:
Thanks for the comments. Just added a "Third-Party Software" table in attr.html. I google searched such format and followed same 3-column format used by other software ([sample here](https://confluence.jetbrains.com/display/IDEADEV/Third-Party+Software+Used+by+IntelliJ+IDEA+15)).